backend/templates/IndexPage.csp

```diff
@@ -97,48 +97,48 @@
 					</el-row>
 				</div>
 			</section>
-
+	
 			<section>
 				<div class="exclusive-line">
 					<h2>Code snippets</h2>
 					<h3>Hello World</h3>
 					<pre><code class="c++">using Callback = std::function&lt;void (const HttpResponsePtr &)&gt; ;
-
-app().registerHandler("/", [](const HttpRequestPtr& req, Callback &&callback)
-{
+	
+	app().registerHandler("/", [](const HttpRequestPtr& req, Callback &&callback)
+	{
 	auto resp = HttpResponse::newHttpResponse();
 	resp->setBody("Hello World");
 	callback(resp);
-});</code></pre>
+	});</code></pre>
```

Comment:
@albaropereyra22 This lack of space is intended. These are C++ shown within a `<pre>` block. Which does not perform space collapsing. When added, theses indents will also show up in the web page.

Please revert the changes within pre blocks.